src/lib.rs

```diff
@@ -992,6 +992,26 @@ mod tests {
         }
     }
 
+    mod bool_output {
+        use super::*;
+
+        #[test]
+        fn success_exit_status_is_true() {
+            assert!(cmd!("true"));
+        }
+
+        #[test]
+        fn failure_exit_status_is_false() {
+            assert!(!cmd!("false"));
+        }
+
+        #[test]
+        #[should_panic]
+        fn io_error_panics() {
+            assert!(cmd!("/"));
+        }
```

Comment:
Nice tests. This might not _always_ be the behavior people want or expect. But it's definitely good to document it in a test.